pyramid_swagger/load_schema.py

```diff
@@ -16,6 +16,7 @@
 
 
 EXTENDED_TYPES = {
+    'number': (float,),
```

Comment:
'float': (float) expected the param type to be 'float', which was not as per swagger spec 1.2 (the format can be float, not the type).